README.md

```diff
@@ -20,104 +38,162 @@ for the following use cases:
 
 Sonobuoy supports Kubernetes versions 1.11, 1.12 and 1.13.
 
-[k8s]: https://github.com/kubernetes/kubernetes
-[e2e]: /docs/conformance-testing.md
-[oview]: https://youtu.be/k-P4hXdruRs?t=9m27s
-
-## More information
-
-[The documentation][docs] provides further information about the conformance
-tests, plugins, etc.
-
-[docs]: https://github.com/heptio/sonobuoy/tree/master/docs
-
 ## Prerequisites
 
 * Access to an up-and-running Kubernetes cluster. If you do not have a cluster,
   we recommend following the [AWS Quickstart for Kubernetes][quickstart] instructions.
 
-[quickstart]: https://aws.amazon.com/quickstart/architecture/heptio-kubernetes/
+* An admin `kubeconfig` file, and the KUBECONFIG environment variable set.
 
-* `kubectl` installed. See [installing via Homebrew (MacOS)][brew] or [building
+* For some advanced workflows it may be required to have `kubectl` installed. See [installing via Homebrew (MacOS)][brew] or [building
   the binary (Linux)][linux].
 
-* An admin `kubeconfig` file, and the KUBECONFIG environment variable set.
+## Installing
 
-[brew]: https://kubernetes.io/docs/tasks/tools/install-kubectl/#install-with-homebrew-on-macos
-[linux]: https://kubernetes.io/docs/tasks/tools/install-kubectl/#tabset-1
+Download one of the releases directly from [here][releases].
 
-## Using the CLI
+Alternatively, you can install the CLI by running:
 
-Sonobuoy also provides a CLI that lets you run Sonobuoy on your cluster. By default, the CLI
-records the following results:
+```
+go get -u -v github.com/heptio/sonobuoy
+```
 
-* Information about your cluster's hosts, Kubernetes resources, and versions.
-* systemd logs from each host. Requires a plugin.
-* The results of a e2e conformance tests.
+Golang version 1.12 or greater is recommended. Golang can be installed via
+[gimme][gimme].
 
-### CLI Prerequisites
+## Getting Started
+To launch Sonobuoy for a short run and wait until it is finished run:
+```
+sonobuoy run --mode quick --wait
```

Comment:
I think this should remove the `-m quick` and just do the regular scan. That can then merge this section with "checking for Cluster Conformance".

Reply:
Did this and removed the quick scan section. Just added a note below this mentioning it as a much shorter alternative for general configuration check.